.github/workflows/elixir.yml

```diff
@@ -0,0 +1,26 @@
+name: Elixir CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup elixir
+      uses: actions/setup-elixir@v1
+      with:
+        elixir-version: 1.9.4 # Define the elixir version [required]
+        otp-version: 22.2 # Define the OTP version [required]
+    - name: Install Dependencies
+      run: mix deps.get
+    - name: Compile Dependencies
+      run: mix deps.compile
+    - name: Run Tests
+      run: mix test
```

Comment:
Please add:
```
name: Check formatting
run: mix format --check-formatted
```

Reply:
Is it correct, that if `mix format --check-formatted` fails (return value > 0), it does not allow to merge?

Reply:
Probably the format check should be the first check?

Reply:
I will add the command. Should be correct. I will test it on my fork.

Reply:
I added the lines